pkg/repository/udmrepo/service/service.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
+)
+
+const (
+	defaultUsername = "default"
+	defaultDomain   = "default"
+)
+
+func Create(logger logrus.FieldLogger) udmrepo.BackupRepoService {
+	///TODO: create from kopiaLib
+	return nil
+}
+
+func GetRepoUser() (username, domain string) {
+	return defaultUsername, defaultDomain
+}
```

Comment:
In one of my functions that needs to get Repo User info, so I must need to import `service` package, but nothing to do with `udmrepo service`, should it be one configuration for `udmrepo` rather than configuration in service?

Reply:
As the discussion, the `service` package means the package to provide all the methods exposed by the unified repo. In this way, it is rational to put the `GetRepoUser` in the `service` package.

However, the package name may be misleading that the package binds to the `BackupRepoService` interface. Therefore:

- Let's add some comments to clarify the usage of the package
- When we see a better name in future, we will change the package name